README.md

```diff
@@ -141,6 +146,9 @@ spec:
 - **objectives\[ \]** *Threshold*, required field, described in [Objectives](#objectives)
   section
 
+- **alertPolicies\[ \]** *AlertPolicy*, optional field, described in [Alert Policies](#alertpolicies)
```

Comment:
I am thinking of a use-case where we have multiple `objectives` per SLO. If we have `alertPollicies` at this level we need to have a way to correlate it with an objective I believe. In order to build an alert practically we need to know data-source, target to calculate burn rates, etc..  Shall we move it under `objectives` or I miss something here?

Reply:
Ah good point, @geototti21, maybe it's worth while to handle this other ticket of splitting up objectives and slis? ticket ##39